app/src/main/java/com/zeapo/pwdstore/utils/PasswordRepository.kt

```diff
@@ -169,7 +169,7 @@ open class PasswordRepository protected constructor() {
             val dir = getRepositoryDirectory()
             // uninitialize the repo if the dir does not exist or is absolutely empty
             settings.edit {
-                if (!dir.exists() || !dir.isDirectory || dir.listFiles()!!.isEmpty()) {
+                if (!dir.exists() || !dir.isDirectory || dir.listFiles()?.isEmpty() == true) {
```

Comment:
When `initialize` is called without storage permissions we'll throw an NPE here, and since I have found myself too angry to actually fix the control flow in `PasswordStore`, I've resorted to 'swallowing' the error here.